addons/tls13-middlebox/bootstrap.js

```diff
@@ -0,0 +1,224 @@
+"use strict";
+
+let {classes: Cc, interfaces: Ci, utils: Cu, results: Cr} = Components;
+
+Cu.import("resource://gre/modules/Preferences.jsm");
+Cu.import("resource://gre/modules/TelemetryController.jsm");
+
+const VERSION_MAX_PREF = "security.tls.version.max";
+const FALLBACK_LIMIT_PREF = "security.tls.version.fallback-limit";
+
+let readwrite_prefs = new Preferences({defaultBranch: true});
+
+// all combination of configurations we care about.
+// for is_tls13 == true we need a server that supports TLS 1.3
+// for is_tls13 == false we need a server that DOES NOT support TLS 1.3
+let configurations = [
+    {max_version: 4, fallback_limit: 4, is_tls13: true, website: "enabled.tls13.com"},
+    {max_version: 4, fallback_limit: 4, is_tls13: false, website: "disabled.tls13.com"},
+    {max_version: 4, fallback_limit: 3, is_tls13: true, website: "tls13.crypto.mozilla.org"},
+    {max_version: 4, fallback_limit: 3, is_tls13: false, website: "control.tls12.com"},
+    {max_version: 3, fallback_limit: 3, is_tls13: true, website: "www.allizom.org"},
+    {max_version: 3, fallback_limit: 3, is_tls13: false, website: "short.tls13.com"}
+];
```

Comment:
I think for now, let's just do:

enabled { version = 4, fallback = 4}
disabled { version = 4, fallback = 4}
control { version = 3, fallback = 3}